### PR TITLE
Pinot 0.6 upgrade and add support for minion component

### PR DESCRIPTION
**Pull request by laxmanchekka**
## Description
Please include a summary of the change, motivation and context.

### Testing
Please describe the tests that you ran to verify your changes. Please summarize what did you test and what needs to be tested e.g. deployed and tested helm chart locally. 

### Checklist:
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules

### Documentation
Make sure that you have documented corresponding changes in this repository or [hypertrace docs repo](https://github.com/hypertrace/hypertrace-docs-website) if required.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 FROM alpine:latest AS builder
 
-ENV PINOT_VERSION=0.5.0
+ENV PINOT_VERSION=0.6.0
```
Comment (aaron-steinfeld) [inline Dockerfile:3]: Please update the service manager to match in `pinot-servicemanager/install.sh` and remove the warning on versions in `pinot-udf/README.md`

Comment (laxmanchekka) [reply Dockerfile:3]: Done. Fixed it.

File: helm/Chart.yaml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
 # This version is same as of base image in Dockerfile. Make sure to update it in both files.
 # Helm chart uses .Chart.Version for image tag.
-appVersion: 0.5.0
+appVersion: 0.6.0
 name: pinot
 type: application
 description: Apache Pinot helm chart
-version: 0.1.1
+version: 0.2.5
```
Comment (laxmanchekka) [inline helm/Chart.yaml:8]:
```suggestion
version: 0.1.1
```

File: helm/templates/controller/prometheusrules.yaml [hunk 3]
```diff
@@ -1,4 +1,4 @@
-{{- if .Capabilities.APIVersions.Has "monitoring.coreos.com/v1/PrometheusRule" }}
+{{- if .Capabilities.APIVersions.Has "monitoring.coreos.com/v1/PrometheusRules" }}
```
Comment (ravisingal) [inline helm/templates/controller/prometheusrules.yaml:1]:
```suggestion
{{- if .Capabilities.APIVersions.Has "monitoring.coreos.com/v1/PrometheusRule" }}
```

**Review by ravisingal — APPROVED** [review]

**Pull request merged**